processing/src/main/java/io/druid/query/QueryRunnerHelper.java

```diff
@@ -64,7 +64,7 @@
               @Override
               public Result<T> apply(Cursor input)
               {
-                log.debug("Running over cursor[%s]", adapter.getInterval(), input.getTime());
+                log.debug("Running over cursor[%s]", input.getTime());
```

Comment:
Strange logging "cursor[input.getTime()]". Also maybe just remove this debugging line